src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java

```diff
@@ -31,6 +31,6 @@ public static void main(String[] args) {
     static int getNumberFullTrucks(int cargo, int carryingCapacity) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return cargo/carryingCapacity;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"